voxblox/include/voxblox/test/layer_test_utils.h

```diff
@@ -76,6 +76,37 @@ class LayerTest {
   static constexpr double kTolerance = 1e-10;
 };
 
+template <typename VoxelType>
+void LayerTest<VoxelType>::CompareVoxel(const VoxelType& voxel_A,
+                                        const VoxelType& voxel_B) const {
+  LOG(FATAL) << "Not implemented for this voxel type!";
+}
+
+template <>
+void LayerTest<EsdfVoxel>::CompareVoxel(const EsdfVoxel& voxel_A,
+                                        const EsdfVoxel& voxel_B) const {
+  constexpr double kTolerance = 1e-10;
+
+  CHECK_NEAR(voxel_A.distance, voxel_B.distance, kTolerance);
```

Comment:
@mfehr not sure what I'm looking at there, looks plausible to me? If that fixes the compilation, SGTM.